src/inverseRobot.js

```diff
@@ -24,7 +25,18 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const inRobot = {};
+
+  const entries = Object.entries(robot);
+
+  for (let i = 0; i < entries.length; i++) {
+    if (inRobot.hasOwnProperty(entries[i][1])) {
+      return null;
```

Comment:
You could make it more simple:
`  for (const entries in robot) {
    if (newRobot.hasOwnProperty(robot[entries])) {
      return null;`

And there would be no need in 'Object.entries(robot)'